modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -27,27 +32,47 @@ def sort(facilities, sort_method)
         when 'home', 'current'
           facilities.sort_by(&:miles)
         when 'alphabetical'
-          facilities.sort_by(&:name)
+          sort_by_name(facilities)
         when 'appointments'
-          sort_by_recent_appointment(facilities)
+          sort_by_recent_appointment(sort_by_name(facilities))
```

Comment:
I wanted to move alpha sorting outside of `sort_by_recent_appointments` because I felt it didn't belong in the method due to its naming